### PR TITLE
mongodb: Fix for Linuxbrew

### PR DESCRIPTION
**Pull request by aphe**
#446 Commenting out option that only affect homebrew for macOS but cause error to linuxbrew.

### CODE REVIEW

File: Formula/mongodb.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def install
     args << "--use-sasl-client" if build.with? "sasl"
     args << "--use-system-boost" if build.with? "boost"
     args << "--use-new-tools"
-    args << "--disable-warnings-as-errors" if MacOS.version >= :yosemite
+    args << "--disable-warnings-as-errors" # if MacOS.version >= :yosemite
```
Comment (rwhogg) [inline Formula/mongodb.rb]: Thanks for the PR!

However, if you don't mind, could you please change this to what Shaun suggested?

> args << "--disable-warnings-as-errors" if !OS.mac? || MacOS.version >= :yosemite

Thanks?

**Comment by sjackman** [comment]
hmm. The bottle failed to upload to Bintray. Perhaps it's too large? The limit is 250 MB (though it can be increased). If you have time, could you please report,
```
brew install --build-bottle mongodb
brew bottle --keep-old mongodb
du -csh mongodb-3.2.6.x86_64_linux.bottle.tar.gz
```

**Comment by sjackman** [comment]
Regardless of the bottle, it did build cleanly, so I'm 👍 for merging this PR whether we get a bottle or not (though I'd prefer a bottle).

**Comment by sjackman** [comment]
Can you please squash the two commits to a single commit?

**Comment by aphe** [comment]
du -csh mongodb-3.2.6.x86_64_linux.bottle.1.tar.gz 
525M	mongodb-3.2.6.x86_64_linux.bottle.1.tar.gz
525M	total

i got merge conflict when try to squash.

**Comment by sjackman** [comment]
Whoah! That bottle is huge! Have a gigabyte. I'm going to merge your patch without the bottle for now.

**Comment by sjackman** [comment]
Merged. Thanks for your contribution, Afriyandi!

**Pull request closed without merging**